### PR TITLE
helm: use Release.Revision to make chart Helm 3 compatible

### PR DESCRIPTION
**Pull request by wtty-fool**

### CODE REVIEW

File: helm/cert-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
       labels:
         {{- include "labels.selector" . | nindent 8 }}
       annotations:
-        releasetime: {{ $.Release.Time }}
+        releaseRevision: {{ $.Release.Revision }}
```
Comment (kopiczko) [inline helm/cert-operator/templates/deployment.yaml:21]:
```suggestion
        releaseRevision: {{ .Release.Revision }}
```

Comment (wtty-fool) [reply helm/cert-operator/templates/deployment.yaml]: Hmm, not according to changes @rossf7 suggested: https://github.com/giantswarm/azure-operator/commit/f2f2e06a9479135c4c0134716e947ba83dd2d384

Comment (kopiczko) [reply helm/cert-operator/templates/deployment.yaml]: I talked with and `$` is a typo.

Comment (rossf7) [reply helm/cert-operator/templates/deployment.yaml]: Both work but according to the docs it should be without the `$`. 

https://helm.sh/docs/chart_template_guide/builtin_objects/

Sorry for the confusion. I'm going to fix it in my previous PRs including in our docs. 😬

Comment (wtty-fool) [reply helm/cert-operator/templates/deployment.yaml]: Thanks, lads. `$` removed, marking it as resolved.

**Pull request merged**